cypher/remove.md

```
@@ -0,0 +1,68 @@
+## Remove a label from a node
+
+To remove a label from a node use the REMOVE clause as follows:
+
+```cypher
+MATCH (n {name: 'Lori'})
+REMOVE n:Admin
+RETURN n.name, labels(n)
+```
+
+Result:
+|n.name|abels(n)|
```

Markdownlint (0.35.0)

Tables should be surrounded by blank lines

(MD058, blanks-around-tables)